consensus/src/aggregation/config.rs

```diff
@@ -0,0 +1,74 @@
+use super::types::{Activity, Epoch, Index};
+use crate::{Automaton, Monitor, Reporter, ThresholdSupervisor};
+use commonware_cryptography::{bls12381::primitives::variant::Variant, Digest};
+use commonware_p2p::Blocker;
+use commonware_utils::{Array, NonZeroDuration};
+use std::num::{NonZeroU64, NonZeroUsize};
+
+/// Configuration for the [Engine](super::Engine).
+pub struct Config<
+    P: Array,
+    V: Variant,
+    D: Digest,
+    A: Automaton<Context = Index, Digest = D>,
+    Z: Reporter<Activity = Activity<V, D>>,
+    M: Monitor<Index = Epoch>,
+    B: Blocker<PublicKey = P>,
+    TSu: ThresholdSupervisor<Index = Epoch, PublicKey = P>,
+> {
+    /// Tracks the current state of consensus (to determine which participants should
+    /// be involved in the current broadcast attempt).
+    pub monitor: M,
```

Comment:
nit: it may not be sufficient to just use the latest participants should be involved?